plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/BoxLayoutProvider.java

```diff
@@ -52,6 +55,9 @@
  */
 public class BoxLayoutProvider extends AbstractLayoutProvider {
     
+    public static final Set<ContentAlignment> DEFAULT_ALIGNMENT =
+            EnumSet.of(ContentAlignment.H_CENTER, ContentAlignment.V_TOP);
+    
```

Comment:
Sorry, but that's not what I meant by algorithm-specific defaults. This is:

https://github.com/eclipse/elk/blob/3e433c9464ab449f9d2a14c6fd5902af9799559b/plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk#L92

That should work with your `EnumSet` as well, I hope. The advantage of doing it this way is twofold: less code, and more complete reference documentation (which documents overridden defaults).

Reply:
I searched for something like this, I will revert the last commit and do it this way.

Reply:
Sorry, I just silently assumed you knew about it. Should have been more precise. 🙂